src/app/profiles/App/Profiles/List/withConnect.ts

```diff
@@ -6,7 +6,7 @@ import { getStatefulContributors } from 'app/profiles/store/selectors';
 import { areContributorsLoading } from 'app/profiles/store/selectors/contributors';
 import { isConnected } from 'app/profiles/store/selectors/connection';
 import { ProfilesState } from 'app/profiles/store/reducers';
-import createMessageSender from 'app/profiles/createMessageSender';
+import createMessageSender from 'webext/createMessageSender';
```

Comment:
Why not move `const receiver = createMessageSender({ id: extensionId });` at root level, or even in `app/profiles/extensionId`, since it’s «static» ?